ts/call-action-form.tsx

```diff
@@ -104,9 +104,11 @@ export class CallActionForm extends React.Component<Props, State> {
 			} as State);
 			this.props.setModal("loading");
 			var campaign = this.getCampaign(this.props.campaignId);
+			var zipField = (document.getElementsByName('input_zip').item(0) as HTMLInputElement);
 			var data = {
 				"campaignId": campaign.id,
-				"userPhone": phone
+				"userPhone": phone,
+				"zipcode": zipField ? zipField.value : ''
```

Comment:
`input_zip` should probably live in the `CallActionForm` state, [as it does in `PetitionForm`](https://github.com/fightforthefuture/battleforthenet/blob/gh-pages/ts/petition-form.tsx#L43), and the architecture to update the zip across components should probably live in [`BFTNFormFlow`](https://github.com/fightforthefuture/battleforthenet/blob/gh-pages/ts/bftn-form-flow.tsx) and look similar to the `setModal` functionality.

Does sending `zipcode` with an empty string in POST data still prompt the user to enter their zipcode, or does it fail with an empty string being an invalid zip?